cmd/cloud/backup.go

```diff
@@ -5,9 +5,10 @@
 package main
 
 import (
-	"github.com/mattermost/mattermost-cloud/internal/tools/utils"
 	"os"
 
+	"github.com/mattermost/mattermost-cloud/internal/tools/utils"
+
```

Comment:
I think this is an extraneous newline, that or the import on line 12 is supposed to be part of the stanza here so that it's grouped with this internal library

Reply:
Apparently both variations are correct from `goimports` perspective. I removed the new line.